LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 JoliCode
+Copyright (c) 2020 JoliCode
```

Comment:
I don't know whether we should update the year every new year or not

Reply:
I remember that the subject had been opened on the Symfony documentation but I do not know the result

Reply:
I think i should revert this modification.
https://github.com/symfony/symfony/pull/35154#discussion_r363029783